### PR TITLE
apply gofmt before writing the router to file.

### PR DESCRIPTION
**Pull request by trusch**
apply gofmt before writing the router to file.

## Checklist:

- [x] The change works as expected.
- [x] New code can be debugged via logs.
- [x] I have added tests to cover my changes.
- [x] I have locally run the tests and all new and existing tests passed.
- [ ] Requires updates to the documentation.
- [ ] I have made the required changes to the documents.

### CODE REVIEW

File: pkg/generators/router/router_test.go [hunk 1]
```diff
@@ -319,6 +319,12 @@ func TestGenerate(t *testing.T) {
 					OperationID: "DELETEAnotherOperationID",
 					Group:       "Another",
 				},
+				PUT: &endpoint{
+					Summary:     "PUT Summary",
+					Description: "PUT Description\nbut a multiline this time",
+					OperationID: "PUTAnotherOperationID",
+					Group:       "Another",
+				},
```
Comment (pragmader) [inline pkg/generators/router/router_test.go:327]: Please add `\n` to one of descriptions in `"/some/path"`, this endpoint is supposed to check the absence of one of the HTTP methods, that's why `PUT` was not here.

File: pkg/generators/router/router_test.go [hunk 2]
```diff
@@ -352,6 +357,9 @@ type AnotherHandler interface {
 	GETAnotherOperationID(w http.ResponseWriter, r *http.Request)
 	// PATCHAnotherOperationID PATCH Description
 	PATCHAnotherOperationID(w http.ResponseWriter, r *http.Request)
+	// PUTAnotherOperationID PUT Description
+	// but a multiline this time
+	PUTAnotherOperationID(w http.ResponseWriter, r *http.Request)
```
Comment (pragmader) [inline pkg/generators/router/router_test.go:362]: This should be removed too.

File: pkg/generators/router/router_test.go (hunk 1 above)
Comment (trusch) [reply pkg/generators/router/router_test.go:327]: the second is also missing a post, but I'll revert this then.

Comment (trusch) [reply pkg/generators/router/router_test.go]: changed.

File: pkg/generators/router/router_test.go (hunk 2 above)
Comment (trusch) [reply pkg/generators/router/router_test.go]: consequently also removed.

**Review by pragmader — APPROVED** [review]

**Pull request merged**